config/tls_cts.go

```diff
@@ -144,6 +146,10 @@ func (c *CTSTLSConfig) Validate() error {
 	}
 
 	// TLS validation
+	if BoolVal(c.Enabled) && ((StringVal(c.Key) == "") || (StringVal(c.Cert) == "")) {
+		return fmt.Errorf("key and cert are required if TLS is enabled")
```

Comment:
nit suggestion - just was realizing that other configs also have tls related fields - it might be helpful in this error message (also below and the warning log) to add some sort of context that its specifically for the cts-tls config.

Reply:
That's a good point, I can update these errors to include that context!